google/resource_container_node_pool.go

```diff
@@ -415,6 +460,37 @@ func nodePoolUpdate(d *schema.ResourceData, meta interface{}, clusterName, prefi
 		}
 	}
 
+	if d.HasChange(prefix + "management") {
+
+		management := &container.NodeManagement{}
+		if v, ok := d.GetOk(prefix + "management"); ok {
+			managementConfig := v.([]interface{})[0].(map[string]interface{})
+			management.AutoRepair = managementConfig["auto_repair"].(bool)
+			management.AutoUpgrade = managementConfig["auto_upgrade"].(bool)
```

Comment:
since this api is `json:omitempty`, you'll also need to add `management.ForceSendFields = []string{"AutoRepair", "AutoUpgrade"}` so you can change values to false

Reply:
Done.